Backend/dev/migrations/provider-dashboard/0007-access-matrix-table-altered.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE atlas_bpp_dashboard.access_matrix ADD user_action_type varchar(255) NOT NULL DEFAULT ''
```

Comment:
Old code will not work with this migration, because we can't even parse this empty value

Reply:
Also you should add migration to both bap and bpp dashboards